atd-etl/cr3_extract_diagram/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10-slim
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y poppler-utils tesseract-ocr
+
+# Copy our own application
+WORKDIR /app
+COPY . /app/cr3_extract_diagram
+
+RUN chmod -R 755 /app/*
+
+# Install requirements
+RUN cd /app/cr3_extract_diagram && apt-get update && \
```

Comment:
I think you can zap the `&& apt-get update` out of this command. There's no need to refresh the debian package manifests in the course of installing python requirements. It's also covered on line 3.

Reply:
thank you!